services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx

```diff
@@ -119,6 +134,25 @@ export const SubmissionSummary = (): React.ReactElement => {
                 setPageLevelAlert('Error fetching the submission. Please try again.')
                 return
             }
+
+            if (submissionAndRevisions.status === 'UNLOCKED') {
+                const unlockedRevision = submissionAndRevisions.revisions.find(rev => rev.revision.unlockInfo)
+                const unlockInfo = unlockedRevision?.revision.unlockInfo
+
+                if (unlockInfo?.updatedBy && unlockInfo?.updatedAt && unlockInfo?.updatedReason) {
+                    //For some reason the type for unlockInfo from graphql is <UnlockedInfo | null | undefined>, not sure where the null came from
```

Comment:
I dont’ know if you still get notified about comments on a closed PR, but pretty much everything in the graphql has null | undefined when things are optional. I don’t remember why